### PR TITLE
Further Esperanto Translation

### PR DESCRIPTION
**Pull request by spagy**

### CODE REVIEW

**Comment by spagy** [comment]
I reverted Dropbox to Dropboks for the sake of consistency

**Comment by djuretic** [comment]
Dankon! =D

File: lang/eo/strings.po [hunk 1]
```diff
@@ -1340,75 +1350,75 @@ msgstr "Malnova&nbsp;Domo"
 
 #: script/world.js:124
 msgid "A&nbsp;Damp&nbsp;Cave"
-msgstr ""
+msgstr "Malseka Kaverno"
 
 #: script/world.js:125
 msgid "An&nbsp;Abandoned&nbsp;Town"
-msgstr ""
+msgstr "Forlastia Urbeto"
```
Comment (djuretic) [inline lang/eo/strings.po:1357]: Small typo (Forlasita), I fixed it myself

File: lang/eo/strings.po [hunk 2]
```diff
@@ -1340,75 +1350,75 @@ msgstr "Malnova&nbsp;Domo"
 
 #: script/world.js:124
 msgid "A&nbsp;Damp&nbsp;Cave"
-msgstr ""
+msgstr "Malseka Kaverno"
```
Comment (djuretic) [inline lang/eo/strings.po:1353]: Those strings need the "&nbsp;" as a separator to keep it in a single line of text. I've corrected it in another commit.

Comment (spagy) [reply lang/eo/strings.po:1353]: Sorry about that, I'm using PoEdit which doesn't display non-breaking spaces any differently to normal spaces so I didn't notice them. I'll make sure to watch out for them.

File: lang/eo/strings.po (hunk 1 above)
Comment (spagy) [reply lang/eo/strings.po:1357]: Thanks :)

File: lang/eo/strings.po (hunk 2 above)
Comment (spagy) [reply lang/eo/strings.po:1353]: Okay that's just completely untrue, PoEdit is showing them. I'm not sure why I missed them out.

Comment (djuretic) [reply lang/eo/strings.po:1353]: Don't worry! And thanks again for your help

**Pull request merged**